docs/user_guide/event_system.md

```diff
@@ -0,0 +1,147 @@
+---
+id: event_system
+title: event_system
```

Comment:
The title should be "Event System". 

<img width="256" alt="image" src="https://github.com/apache/yunikorn-site/assets/26764036/7764f45d-147b-428a-9091-6c16d3e6e143">